platform/engineering/design-docs/flag-for-manual-review.md

```diff
@@ -0,0 +1,112 @@
+# Flag for manual review process
+
+**Author(s):** Brooks Johnson (brooks@adhoc.team)
+**Last Updated:** February 20, 2020  
+**Status:** **Draft** | In Review | Approved  
+**Approvers:** _Rian Fowler_ \[ \], _Andrew Gunsch_ \[ \], ...  
+
+
+
+## Overview
+
+### Objective
+
+The goal is to improve our review process which by adding automated processes which will alert a review team when a manual review is required.
+These changes are meant to work alongside the Codeowners changes which will give each app team primary responsibility for reviewing their code.
+
+
+Intended audience: VSP team
+
+### Background
+
+From the perspective of a VFS engineer, the current review process involves:
+
+1. Creating a Draft PR with appropriate changes
+1. Getting a peer review
+    - Making any changes until the review is approving
+1. Marking the PR as `Ready for Review`
+1. Getting approval from a member of the VSP *frontend-review-group*
+
+The final approval from the review group is often not helpful, and it slows the process down.
+This frustration often leads to VFS teams ignoring this process and they will directly ask for a VSP review before their PR is ready.
```

Comment:
I avoided that because that felt like part of the solution, and the template said to [keep that out of the background](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/design-docs/design-doc-template.md#background).  I'll try to phrase it in a way that keeps the emphasis on the problem and not the solution.